### ISSUE TITLE
Chrome not automatically closing when running script via guestcontrol in VM 

### ISSUE THREAD

**Issue by JosefLeinweber** [aisingapore/TagUI#1416]
I have a VirtualBox VM with windows installed. 
When I use guestcontrol to start a tagui script in the VM:
``` 
VBoxManage guestcontrol "test2" run --exe C:\Windows\System32\cmd.exe --username <username> --password <password> -- /c "tagui C:\Users\something\Documents\tagUIscripts\testing.tag"
```
The script gets executed successfully and outputs `FINISH - automation finished - <time>` but it does not return.
But chrome does not close automatically afterwards and the guestcontrol command does not return. Only after manually closing chrome in the VM the guestcontrol command returns. 

I have updated tagui via `tagui update`
This is the test script I executed:
```
echo hello world
keyboard [win]d
```

I'm not sure if this is only a tagui issue, but I thought it's better to point it out.  Maybe it's also just a silly mistake from my side, but I can't make it work.

**Comment by JosefLeinweber** [comment]
An easy workaround is to append ` && taskkill /IM chrome.exe` to the guestcontrol command. This makes the guestcontroll command return successfully.

**Comment by kensoh** [comment]
Thanks @JosefLeinweber for sharing back! There is already killing of Chrome after TagUI finishes running.

Maybe for some reason it can't work in your setup above. Below is the line that kills Chrome.

https://github.com/aisingapore/TagUI/blob/37a28b5196fd3a6bc88c1fb83bc3ba8f001c3cc0/src/tagui.cmd#L1199

**Comment by JosefLeinweber** [comment]
Yes, it does not work in my setup, but I don't know why. If I figure it out, I will let you know.

**Comment by kaensoft** [comment]
Same issue on a VirtualBox guest win11, chrome is still open when the task has finished.

Possibly the issue is related to WMIC, as Windows 11 24H2 has WMIC disabled by default.
https://techcommunity.microsoft.com/blog/windows-itpro-blog/wmi-command-line-wmic-utility-deprecation-next-steps/4039242

Looks like the code line in tagui.cmd "wmic process where "caption like '%%chrome.exe%%' and commandline like '%%tagui_user_profile_ --remote-debugging-port=9222%%' and not commandline like '%%--type=renderer%%'" get processid 2^>nul ^| cut -d" " -f 1 ^| sort -nur ^| head -n 1" could not get the correct chrome processid as expected.

<img width="825" alt="image" src="https://github.com/user-attachments/assets/7c8668da-8426-4752-9dee-2469951abe62">


Added WMIC back via System->Optional features.
It works for me now, chrome is closed by default as expected.

Hope it helps.

**Comment by kensoh** [comment]
Thanks @kaensoft for sharing your solution!

Yes wmic is used to get the process id of the web browser to kill it at the end of the automation. With wmic disabled, browser won't close.
